rfs_test/TEST_accountservice_schema.py

```diff
@@ -18,283 +18,470 @@
             except:
                 assertion_status = log.WARN
-                log.assertion_log('line', "~ \'Members\' not found in the payload from GET %s" % ('/redfish/v1/AccountService'))
+                log.assertion_log('line', "~ \'Members\' not found in the payload from GET %s" % (
+                    '/redfish/v1/AccountService'))
                 return assertion_status
             else:
                 for member in members_collection:
                     key = '@odata.id'
                     member_link = member[key]
-                    json_payload, headers, status = self.http_PUT(member_link, rq_headers, rq_body, authorization)
+                    json_payload, headers, status = self.http_PUT(
+                        member_link, rq_headers, rq_body, authorization)
                     if status == 201:
                         assertion_status = log.FAIL
                         log.assertion_log(assertion_status, None)
-                        log.assertion_log('line', "Assertion Failed")  
+                        log.assertion_log('line', "Assertion Failed")
                         return assertion_status
-            
-    log.assertion_log(assertion_status, None)
-    log.assertion_log('line', "Assertion Passes") 
+
+    log.assertion_log(assertion_status, log.PASS)
+    log.assertion_log('line', "Assertion Passes")
 
     return assertion_status
-    
-## end Assertion_ACCO101
+
+# end Assertion_ACCO101
 
 ###################################################################################################
-# Name: Assertion_ACCO102(self, log) :Account Service                                            
-# Assertion text: 
-# This property shall reference the threshold for when an authorization failure is logged.  This 
-#represents a modulo function value, thus the failure shall be logged every nth occurrence where 
-#n represents the value of this property.
+# Name: Assertion_ACCO102(self, log) :Account Service
+# Assertion text:
+# This property shall reference the threshold for when an authorization failure is logged.  This
+# represents a modulo function value, thus the failure shall be logged every nth occurrence where
+# n represents the value of this property.
 ###################################################################################################
 
-'''
-Step 1: Simulate an authorization failure by providing wrong credentials.
-Step 2: Check at which attempt the authorization failure is logged.
-Step 3: Compare the value obtained from STEP 2, with the value extracted from the key "minimum" of 
-the property AuthFailureLoggingThreshold in the JSON Schema file.
-Step 4: Fail the assertion if the values at STEP 3 does not match.
 
-Concerns: What if the value AuthFailureLoggingThreshold is very large, thus requiring a considerable 
-amount of time to test the assertion. 
-'''
-## end Assertion_ACCO102
+def Assertion_ACCO102(self, log):
```

Comment:
Unfortunately I don't know how to programmatically test for this; the challenge is you need a priori knowledge about the specific log that will be generated when this condition happens (and which log for a particular manager will contain the entry).

Reply:
Removed for the list of assertions.